src/main/java/org/sagebionetworks/web/client/Portal.java

```diff
@@ -72,6 +73,28 @@ public void onSuccess() {
               // make sure jsni utils code is available to the client
               ginjector.getSynapseJSNIUtils();
 
+              if (DisplayUtils.isInTestWebsite(ginjector.getCookieProvider())) {
+                ginjector
+                  .getSynapseJavascriptClient()
+                  .getFeatureFlagConfig(
+                    new AsyncCallback<String>() {
+                      @Override
+                      public void onSuccess(String config) {
+                        ginjector
+                          .getCookieProvider()
+                          .setCookie("PORTAL_FEATURE_FLAG", config);
+                      }
+
+                      @Override
+                      public void onFailure(Throwable reason) {
+                        _consoleError(
+                          "Error getting configurations:" + reason.getMessage()
+                        );
```

Comment:
Just being picky about making the error message clear. We'll have to work through how we actually handle failure cases when we retrieve a specific flag, but what you have is good enough for now.
```suggestion
                        _consoleError(
                          "Error getting feature flag configuration:" + reason.getMessage()
                        );
```